workflow/comp_pred_ridge_and_LSTM.ipynb

```diff
@@ -0,0 +1,2196 @@
YwXWAOrKZl0xue/F2xGfFQ7yrRVL4IyFfEbqCSRCRiVlcAhQbG4uvv/4au3fvhp+fX4Nlb926hbZt22Lt2rV47rnnGv0aHTp0wKJFizBt2rR7lmUCZOa0GjbzkqQ0Wg0ivozQa/m5kwABCicFMsZlsDuMyISa8vkt6RggURQxY8YMpKWlISsr657JT+1zRFHU67K6lwsXLuDKlSvw8vK6d+EWxLECRlI7W4RIIkfLjtab/ACACBGlFaU4WnYUA5UDTRgZETWWpAlQTEwMNm7ciK+//hqurq4oLS0FALi7u8PR0RG//PILNm3ahPDwcHTo0AEXLlzAkiVL4OjoiCeeeEJXT8+ePZGUlISxY8fi5s2bSExMxLhx46BUKlFYWIi5c+fC398fERERUt0qxwoQtSKXKy4btRxZALY8tzqSJkDJyckAgNDQUL3zKSkpiI6OhoODA/bs2YOVK1fi2rVrUCgUGDp0KPbt2we5XK4rn5+fr5tBJpPJcOzYMXz66ae4fv06vL29ER4ejnfeeQf29vYmu7c71TdWoKyiDPFZ8RwrQGRhOjh1MGo5MnPcg7BVMosxQObGmGOAOFaAqPWp/XddVlFW54sNwH/XrYpu9undP2fOPjVHTfn85l5gLawpYwWIyDLIbGSYFzwPwJ+zvmrVHicEJzD5sXTcg7BVYwLUwjhWgKh1CvMNw4rQFZA7yfXOK5wU7NZuLbgHYatmVitBt0YcK0DUeoX5hmGYzzDO7mytuAdhq8YEqIUFyYOgcFLcc6xAkDxIguiI6H7JbGSc6t5acQ/CVo1dYC2MYwWIiCxU7R6EdXZZryXUrETPPQgtEhMgE+BYASIiC2Qjq5nqDqBuEvTHceQSrgdkoTgN3oCW2gqDK0ETEVkg7kFoMSxuLzBzw73AiIhID1eCtggWsxcYERGRReAehK0OxwARERGR1WECRERERFaHCRARERFZHSZAREREZHWYABEREZHVYQJEREREVocJEBEREVkdJkBERERkde5rIcTKykps2rQJ5eXlGD58OLp162asuIiIiIhaTKMToPj4eFRXV+P9998HANy6dQshISH46aef4OTkhLlz5yIzMxMhISEtFiwRERGRMTS6C+z777/H8OHDdccbNmxAcXExCgoKcO3aNTzzzDNYtGhRiwRJREREZEyNToDOnTuHgIAA3fH333+Pp59+Gr6+vhAEAbNmzcIPP/zQIkESERERGVOjEyAbGxvcuXH8gQMH8PDDD+uOPTw8cO3aNeNGR0RERNQCGp0A9erVC9988w0A4KeffsK5c+cwbNgw3fXi4mIoFArjR0hERERkZI0eBD137lw8++yz+N///oeffvoJTzzxBPz8/HTXv/32WwQHB7dIkERERETG1OgWoLFjx+Lbb79F37598eqrr2LTpk16152cnPDKK68YPUAiIiIiYxPEOwf2NODtt9/GnDlz4OTk1NIxSU6tVsPd3R0qlQpubm5Sh0NERESN0JTP70a3ACUmJuLmzZv3HRwRERGR1BqdADWyoYiIiIjI7DVpLzBBEFoqDiIiIiKTadJeYN27d79nEnT16tX7CoiIiIiopTUpAUpMTIS7u3tLxUJERERkEk1KgJ599lnI5fKWioWIiIjIJBo9Bojjf4iIiKi14CwwIiIisjqNToC0Wq3Ru7+SkpIwcOBAuLq6Qi6XIyoqCvn5+Xplpk2bhq5du8LR0REdOnTAmDFjcPr06QbrFUURb731Fry8vODo6IiwsDAUFBQYNXYiIiKyXE2aBm9s2dnZiImJwYEDB5CZmYnq6mqEh4ejvLxcV6Z///5ISUnBqVOnsG3bNoiiiPDwcGg0mnrrXbZsGVavXo01a9YgNzcXzs7OiIiIQGVlpSlui4iIiMxco7fCMIXLly9DLpcjOzsbQ4cONVjm2LFjePDBB3HmzBl07dq1znVRFOHt7Y3Zs2djzpw5AACVSgWFQoHU1FQ8++yz94yDW2GYN41Wg6NlR3G54jI6OHVAkDwIMhuZ1GEREZHEmvL53aRZYC1NpVIBADw9PQ1eLy8vR0pKCvz8/ODj42OwTFFREUpLSxEWFqY75+7ujkGDBmH//v0GE6CqqipUVVXpjtVq9f3cBrWg7cXbseTgElyquKQ7p3BSYF7wPIT5hjXwTCLrxS8NRHVJ2gV2J61Wi7i4OAwZMgSBgYF61z744AO4uLjAxcUF3333HTIzM2FnZ2ewntLSUgCAQqHQO69QKHTX7paUlAR3d3fdo77kiqS1vXg74rPi9ZIfACirKEN8Vjy2F2+XKDIi87W9eDsivozAlG1TkLAnAVO2TUHElxH890JWz2wSoJiYGJw4cQKff/55nWsTJkzADz/8gOzsbHTv3h3jx4836nie+fPnQ6VS6R7nz583Wt1kHBqtBksOLoGIuj22teeWHlwKjbb+sWFE1oZfGojqZxYJUGxsLLZu3Ypdu3ahU6dOda67u7ujW7duGDp0KL744gucPn0aaWlpButSKpUAgEuX9P/BX7p0SXftbvb29nBzc9N7kHk5Wna0zn/idxIhorSiFEfLjpowKiLzxS8NRA2TNAESRRGxsbFIS0vDzp074efn16jniKKoN2bnTn5+flAqldixY4funFqtRm5uLkJCQowWO5nW5YrLRi1H1NrxSwNRwyRNgGJiYrB+/Xps3LgRrq6uKC0tRWlpKX7//XcAwC+//IKkpCQcOXIE586dw759+/DMM8/A0dERTzzxhK6enj176lqEBEFAXFwcFi1ahPT0dBw/fhyTJk2Ct7c3oqKipLhNMoIOTh2MWo6oteOXBqKGSToLLDk5GQAQGhqqdz4lJQXR0dFwcHDAnj17sHLlSly7dg0KhQJDhw7Fvn379BZlzM/P180gA4C5c+eivLwcU6dOxfXr1/HII48gIyMDDg4OJrkvMr4geRAUTgqUVZQZbNIXIEDhpECQPEiC6IjMD780EDXMrNYBMhcttQ6QRiviYNFVlN2ohNz
```

Comment:
Again it would be nice to see the performance of the mean here. Technically it should be the mean of the training data for a more fair comparison.

Reply:
I would think something goes wrong then. If the MSE is 97, that means that the non-squared error is 9.8 K. 

I would expect the mean of the training to be much more like the mean of the ground truth, as in this image:
![image](https://github.com/AI4S2S/cookbook/assets/12114825/26571aa2-64fc-4d12-8fb0-aa7dad724d84)

The validation SST varies between 22 and 25 degrees C. With that small variance I would not expect to ever get a MSE of 97.

Reply:
😂 I thought it was because of the trend. For a fair comparison, we took the mean of the data preceding the test period (about 30 years). And 30 years of trend causes the big error.

Anyway, it is better to use climatology rather than the mean. This is fixed in another PR #11 by Jannes. Thanks for checking it.